Example/Auth/Tests/FIRPhoneAuthProviderTests.m

```diff
@@ -721,6 +731,70 @@ - (void)testVerifyPhoneNumberUIDelegateUnexpectedError {
   OCMVerifyAll(_mockNotificationManager);
 }
 
+/** @fn testVerifyPhoneNumberUIDelegateUnstructuredError
+    @brief Tests a invocation of @c verifyPhoneNumber:UIDelegate:completion: which results in an
+        invalid client ID.
```

Comment:
... which results in an error being surfaced due to an unexpected structure of the error response.